src/core/rendering.js

```diff
@@ -1247,7 +1247,7 @@ p5.prototype.blendMode = function(mode) {
  *   // Draw the circle.
  *   circle(50, 50, 40);
  *
- *   describe('A fiery sun drawn on a light blue background.');
+ *   describe('A fiery sun drawn on a light #1a85ff background.');
```

Comment:
```suggestion
 *   describe('A fiery sun drawn on a light blue background.');
```

Maybe the last bit. This changes you have made on the `drawingContext` method. So this can be reverted to it's color name.